modules/govuk_bundler/manifests/config.pp

```diff
@@ -0,0 +1,32 @@
+# == Define: govuk_bundler::bundler
+#
+# Manage the config used by bundler for access to gemstash.
+#
+# === Parameters
+#
+# [*user_home*]
+#   The home directory of the user to add the config to.
+#   Default: /home/deploy
+#
+# [*server*]
+#   The gemstash server to use
+#   Default: http://gemstash.cluster
+#
+define govuk_bundler::config(
+  $user_home = '/home/deploy',
```

Comment:
If this is a generic define to be reused in different classes, I don't know if it's better to make the user_home mandatory, so we don't make any assumptions about a specific user